tests/virt_autotest/guest_installation_run.pm

```diff
@@ -86,6 +86,19 @@ sub run {
         $upload_guest_assets_flag = 'yes';
     }
 
+    if (check_var('BOOT_64KB_PAGE_SIZE', '64kps')) {
+        # Verify 64kb page size enabled.
+        record_info('Baremetal kernel cmdline', script_output('cat /proc/cmdline'));
+        assert_script_run("grep -- '-64kb' /proc/cmdline");
```

Comment:
Maybe you can search dmesg or /var/log/message to determine it is running 64kb kernel ? It is up to you to pick the better one.

Reply:
I think it can 100% judge kernel 64kb boot page size.